custom_components/tahoma/climate_st.py

```diff
@@ -99,6 +102,15 @@ def __init__(
     async def async_added_to_hass(self):
         """Register temperature sensor after added to hass."""
         await super().async_added_to_hass()
+
+        # The Somfy Thermostat require a temperature sensor
+        base_url = self.device.deviceurl.split("#", 1)[0]
+        entity_registry = await self.hass.helpers.entity_registry.async_get_registry()
+        for k, v in entity_registry.entities.items():
```

Comment:
what is `k` and `v`? (besides key/value). Would be good to add a bit more meaningful names to improve readability. Since it is not a plain key value list, but it is reflecting a device?

Reply:
for loop removed following next comment.